make.py

```diff
@@ -21,7 +21,11 @@
     path = path / file_
     if not path.exists():
         params = {'data': f'[out:json];relation{tags};(._;>;);out body;'}
-        resp = requests.get(OVERPASS, params=params)
+        try:
+            resp = requests.get(OVERPASS, params=params)
+            resp.raise_for_status()
+        except requests.exceptions.HTTPError as error:
+            raise ValueError(f'The request to overpass API failed: {error.response.status_code}')
```

Comment:
Why not using `resp.status_code` directly here?

Reply:
Because I am not familiar with this, and used an example found in documentation

Reply:
I changed it a bit following https://stackoverflow.com/questions/16511337/correct-way-to-try-except-using-python-requests-module, what do you think of: https://github.com/tilery/mae-boundaries/pull/7/commits/0b3656584beda08a02384bd6a2c30aeddfd3aa1c

But I think we should retry it after a while if we have a 429 HTTP error if we want to lauch this script with CI.

Maybe in an other enhancement report?